pvanalytics/quality/gaps.py

```diff
@@ -125,6 +125,57 @@ def stale_values_diff(x, window=6, rtol=1e-5, atol=1e-8, mark='tail'):
     return _mark(flags, window, mark)
 
 
+def stale_values_round(x, decimals=3, window=6, mark='tail'):
+    """Identify stale values by rounding.
+
+    A value is considered stale if it is part of a sequence of length
+    `window` of values that are identical when rounded to `decimals`
+    decimal places.
+
+    Parameters
+    ----------
+    x : Series
+        Data to be processed.
+    decimals : int, default 3
+        Number of decimal places to round to.
+    window : int, default 6
+        Number of consecutive identical values for a data point to be
+        considered stale.
+    mark : str, default 'tail'
+        How much of the window to mark ``True`` when a sequence of
+        stale values is detected. Can be of 'tail', 'end', or 'all'.
+
+        - If 'tail' (the default) then every point in the window
+          *except* the first point is marked ``True``.
+        - If 'end' then only the endpoints of the window are marked
```

Comment:
only the last point? I'm confused by "endpoints"

Reply:
This is basically the same as the default behavior for `pandas.Series.rolling` where the value is assigned to the rightmost label in the window. Maybe it would be better to say 'If end then the first `window-1` values in a sequence of stale values are marked False, and all subsequent values in the sequence are marked true'.